overlays/server-os-debian/etc/initramfs-tools/scripts/init-premount/stiefellink

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
```

Comment:
this script does more than just initialize the stiefellink; the filename should be just `stiefelsystem`